src/generator/gen_unicode.ml

```diff
@@ -46,7 +46,7 @@ let print_elements ch hashtbl cats =
           (fun (b, e) -> Printf.sprintf "0x%x, 0x%x" b e)
           (Cset.union_list (Hashtbl.find_all hashtbl c) :> (int * int) list)
       in
-      Printf.fprintf ch "  let %s = Sedlex_cset.of_list\n    [" c;
+      Printf.fprintf ch "  let %s = Sedlex_utils.Cset.of_list\n    [" c;
```

Comment:
I don't understand why we need this diff

Reply:
`unicode.ml` needs to compilable from within `src/syntax` and as `examples/unicode_old.ml`. 

Ideally, `unicode.ml` should be copiable to `unicode_old.ml` without any chances.

If you use `Sedlex_ppx.Sedlex_cset` is does not compile in `src/syntax`.

Happy to find a better solution but, also, I don't think that it matters.

Reply:
Ok reverted.